bemuse/tasks/build.js

```diff
@@ -1,20 +1,21 @@
-import Promise from 'bluebird'
+import buildConfig from '../config/buildConfig'
+import config from '../config/webpack'
 import fs from 'fs'
 import gulp from 'gulp'
 import log from 'fancy-log'
-import webpack from 'webpack'
-
-import buildConfig from '../config/buildConfig'
-import config from '../config/webpack'
 import path from '../config/path'
+import webpack from 'webpack'
 
-const readFile = Promise.promisify(fs.readFile, fs)
-const writeFile = Promise.promisify(fs.writeFile, fs)
+const { readFile, writeFile } = fs.promises
 
 gulp.task(
   'build',
   gulp.series('dist', async function () {
-    const stats = await Promise.promisify(webpack)(config)
+    const stats = await new Promise((resolve, reject) =>
+      webpack(config, (err, stats) =>
+        err === null ? resolve(stats) : reject(err)
```

Comment:
The way to check for error in webpack as documented is [to check for truthiness or falsiness](https://webpack.js.org/api/node/#webpack). To align with the docs, check for `!err` instead of `err === null`.

Reply:
Fixed on `a0ff43bc`.